ocsp/responder.go

```diff
@@ -148,6 +147,7 @@ type Responder struct {
 	responseTypes *prometheus.CounterVec
 	requestSizes  prometheus.Histogram
 	clk           clock.Clock
+	log           blog.Logger
```

Comment:
Do you need to initialize this in NewResponder (line 177) as well?

Reply:
Uh, yes, whoops.